app/scripts/modules/core/src/projects/configure/ProjectAttributes.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import { FormikErrors } from 'formik';
+import { IProject } from 'core/domain';
+import { IWizardPageProps, wizardPage } from 'core/modal';
+import { FormField, FormikFormField, TextInput } from 'core/presentation';
+
+export interface IProjectAttributesProps extends IWizardPageProps<IProject> {
+  onDelete?: Function;
+  allProjects: IProject[];
+}
+
+interface IProjectAttributesState {
+  showProjectDeleteForm: boolean;
+  projectNameForDeletion: string;
+}
+
+class ProjectAttributesImpl extends React.Component<IProjectAttributesProps, IProjectAttributesState> {
+  public static LABEL = 'Project Attributes';
+
+  public state = {
+    showProjectDeleteForm: false,
+    projectNameForDeletion: null as string,
+  };
+
+  public validate(values: IProject) {
+    const errors: FormikErrors<IProject> = {};
+
+    const isValidName = (name: string) => {
+      const namePattern = /^[^\\\^/^?^%^#]*$/;
+      return name.match(namePattern);
+    };
+
+    const isValidEmail = (email: string) => {
+      const emailPattern = /^(.+)\@(.+).([A-Za-z]{2,6})/;
```

Comment:
Not in this PR, but we really need to pull these types of validators into a utility.